johnsonandjohnson/Bodiless-JS
# feat(core): Preview Mode Session

## Changes
- Edit mode is now persisted with a sessionStorage

## Test Instructions

- Run `npm run start`
- Switch the edit mode off.
- Navigate off the page or refresh it. The edit mode should stay off.

## Related Issues
Closes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/114
